test/integration/publish_int_test.go

```diff
@@ -462,6 +466,8 @@ authors:
 			}
 		})
 	}
+
+	ts.IgnoreLogErrors()
```

Comment:
What is this ignoring? Please add a comment if we truly want this.

Reply:
There are test cases that test failures, like omitting a filename or being unable to edit a description. These expected failures will be written to the log and cause nightly integration failures.

We have similar precedent for other integration tests that induce failures and verify that they actually do fail (and gracefully).